index.d.ts

```diff
@@ -10,4 +10,6 @@ import {Chai} from './lib/Chai';
 
 declare var chai: Chai;
 
-export = chai;
+export default chai;
```

Comment:
I think this can be merged it you don't use this and remove the `Chai` interface instead actually. cc @unional, does that sound right?